README.md

```diff
@@ -56,6 +56,8 @@ run style checks use `composer cs`.
 #### Other changes
    
 * Changed the minimum PHP version to 7.0
+* Added `source` field to `subscriptions` table. This field indicates what led to the description,
```

Comment:
Sorry, should've done a PR myself. I meant "This field indicates what led to the **sub**scription ..."